ui/src/scss/mixins.scss

```diff
@@ -49,3 +49,9 @@
     transform: translateX(-100%);
   }
 }
+
+%handel-text-overflow {
```

Comment:
Typo in the name. I am approving/merging the PR; try to keep in mind to fix this whenever it is used next time.